sdk/nodejs/tests/runtime/testdata/closure-tests/test.ts

```diff
@@ -57,8 +57,12 @@ it(`resolve to the correct typescript version within the pulumi package`,
 describe(`closure tests (TypeScript ${typescript.version})`, function () {
     const cases = readdirSync("cases"); // describe does not support async functions
     for (const testCase of cases) {
-        const { func, isFactoryFunction, error: expectedError, description, allowSecrets, after } = require(`./cases/${testCase}`);
-
+        const filesInCase = readdirSync(`./cases/${testCase}`);
+        const testCaseFile = filesInCase.find((file: string) => /^index.(cj|j)s$/.test(file));
+        if (!testCaseFile) {
+            throw new Error(`No test file specified for case ${testCase}`);
+        }
```

Comment:
I'm still a bit wary of this -- the regex doesn't match `.ts` files, right? Doesn't that mean we'll throw errors for several legitimate cases? Apologies if I'm being stupid and really missing the point here :facepalm:

Reply:
Thanks for explaining this -- makes total sense! And you are right about CI hopefully having our backs here. I've kicked off a test run; hopefully we can get this shipped!